config/annotations.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const Joi = require('joi');
+
+// List of data, eventually key-value pairs
+const SCHEMA_ANNOTATIONS = Joi.object().optional();
```

Comment:
I had based `annotations` off of how `workflow` was implemented. If there is a preference to define everything locally instead of its own class, I'm amenable to that.